src/pages/resultsView/network/network.htm

```diff
@@ -12,11 +12,14 @@
 
     var queryParams = url("?");
 
-    var host = "//" + window.location.hostname + ":" +  window.location.port  +  "/reactapp/network/files";
-
-    host = "./files";
-
-    var apiHost = "//" + queryParams.apiHost;
+    var host = "./files";
+    var apiHost;
+    // if this fails, we are NOT in production, so allow anything
+    try {
```

Comment:
ino, the logic here is that when we are in production the frame will always be hosted on same domain as parent, so location will be accessible and we can use it as apiHost.  Only in dev will iframe be cross domain, and in that case, we can accept a domain passed in from qstring.